models/award.ts

```diff
@@ -10,144 +10,144 @@ export class Award {
   fiscal_code: FiscalCode;
 
   @ViewColumn({
-    name: "aw_winn_award_period_id_n"
+    name: "id_n"
   })
-  aw_winn_award_period_id: number;
+  award_winner_id: number;
```

Comment:
Can we add even `id_n` that is the id of the `award_winner`? Could be useful as information.

Reply:
Sorry, that field was added with this PR, we can add `aw_winn_award_period_id_n` that is the id of the period.

Reply:
@BurnedMarshal is it ok for you?

Reply:
@pasqualedevita  lgtm. What I mean is that we can add even the field `aw_winn_award_period_id_n` to have the `period_id` useful for debugging purpose.

Reply:
ok sorry, you are right. added award_winner_period_id